### PR TITLE
Functional tests - Update puppeteer to version 3.0.2

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Update puppeteer to version 3.0.2 to get the newest chromium binary and to experiment firefox (before full support)
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | No test needed (already tested)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18964)

### CODE REVIEW

**Review by matthieu-rolland — DISMISSED** [review]

File: tests/puppeteer/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "lint": "eslint  --fix --ignore-path .gitignore .",
     "specific-test": "./node_modules/mocha/bin/mocha campaigns/$TEST_PATH",
     "sanity-tests": "./node_modules/mocha/bin/mocha campaigns/sanity/*",
-    "sanity-travis": "./node_modules/mocha/bin/mocha --bail campaigns/sanity/*",
+    "sanity-travis": "node --version && ./node_modules/mocha/bin/mocha --bail campaigns/sanity/*",
```
Comment (PierreRambaud) [inline tests/puppeteer/package.json]: Why checking node version here?

Comment (boubkerbribri) [reply tests/puppeteer/package.json]: I debug, I have a problem only on travis 
image
and I think its linked to node version
puppeteer v3 support node 10 and we have only node 8 on travis.
I don't know if we can change it ?

Comment (boubkerbribri) [reply tests/puppeteer/package.json]: I tried to use nvm but not found, @PierreRambaud can you tell me we can switch to node 10 on travis ?

Comment (PierreRambaud) [reply tests/puppeteer/package.json]: A possible way: https://medium.com/@_Ma27_/travis-ci-with-php-and-nodejs-done-right-57bd4d650f75
We are using the PHP container, you need to install nodejs manually :/

Comment (boubkerbribri) [reply tests/puppeteer/package.json]: Thanks @PierreRambaud , it works

File: travis-scripts/setup-node-js.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+echo "Install node js v10 for puppeteer v3";
+
+sudo rm -rf ~/.nvm
+curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
```
Comment (PierreRambaud) [inline travis-scripts/setup-node-js.sh:6]: We should install nodejs for the whole project. Wdyt? 
ping @NeOMakinG

File: .travis.yml [hunk 3]
```diff
@@ -72,6 +72,7 @@ before_install:
   # Apache & php-fpm configuration
   - bash travis-scripts/setup-php-fpm.sh
   - bash travis-scripts/setup-apache.sh
+  - if [ "$PRESTASHOP_TEST_TYPE" = "sanity" ]; then bash travis-scripts/setup-node-js.sh; fi;
```
Comment (PierreRambaud) [inline .travis.yml]:
```suggestion
  - bash travis-scripts/setup-node-js.sh;
```

File: travis-scripts/setup-node-js.sh [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+echo "Install node js v10 for puppeteer v3";
```
Comment (PierreRambaud) [inline travis-scripts/setup-node-js.sh]:
```suggestion
echo "Install Node.js";
```

File: .travis.yml [hunk 5]
```diff
@@ -72,6 +72,7 @@ before_install:
   # Apache & php-fpm configuration
   - bash travis-scripts/setup-php-fpm.sh
   - bash travis-scripts/setup-apache.sh
+  - bash travis-scripts/setup-node-js.sh;
```
Comment (PierreRambaud) [inline .travis.yml]:
```suggestion
  - bash travis-scripts/setup-node-js.sh
```

**Review by PierreRambaud — APPROVED** [review]
:heart:

**Review by Progi1984 — APPROVED** [review]

**Review by SimonGrn — APPROVED** [review]

**Pull request merged**